docs/pages/video.js

```diff
@@ -1,192 +1,334 @@
-        description={`
-    \`Video\` supports the native HTML video attributes such as \`autoPlay\`, \`loop\`, \`muted\`, and more.
-    Simply pass these through as props on the \`Video\` component.
-  `}
-        defaultCode={`
-<Video
-  aspectRatio={1920 / 1080}
-  captions=""
-  loop
-  playing
-  src="https://media.w3.org/2010/05/sintel/trailer_hd.mp4"
-/>
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection
+          title="Labels"
+          description={`Video requires several accessibility labels for each video control: \`accessibilityMaximizeLabel\`, \`accessibilityMinimizeLabel\`, \`accessibilityMuteLabel\`, \`accessibilityPauseLabel\`, \`accessibilityPlayLabel\`, \`accessibilityProgressBarLabel\` and \`accessibilityUnmuteLabel\`.
+
+If the video contain captions, it also requires \`accessibilityHideCaptionsLabel\` and \`accessibilityShowCaptionsLabel\`.
+.
 `}
-      />
-      <Example
-        id="videoControlsExample"
-        name="Video controls"
-        description={`
-    \`Video\` components can show a control bar to users in order to allow them access to certain features
-    such as play/pause, timestamps, mute, and fullscreen. Pass in the \`controls\` prop to make them appear.
+        />
+      </MainSection>
+
+      <MainSection name="Variants">
+        <MainSection.Subsection
+          title="Autoplay and error detection"
+          description={`Autoplay or automatically starting the playback of the video requires the \`autoplay\` prop. While autoplay of media serves a useful purpose, it should be used carefully and only when needed. In order to give users control over this, browsers often provide various forms of autoplay blocking.
+
+Autoplay blocking is not applied to video elements when the source media doesn't have an audio track or is muted.
+
+Gestalt Video provides a comprehensive API to handle gracefully autoplay blocking and prevent UI and/or client errors. The example below shows how to correctly handle autoplay error detection.
+
+If \`autoplay\` is set, use \`onPlay\` to detect when the video starts playing and display the pause control accordingly. In case \`autoplay\` gets blocked, \`onPlay\` would never get triggered and controls will still display the play control.
+
+\`onPlayError\` is another error-handling callback. In this case, \`onPlayError\` detects if the HTMLMediaElement.play() method fails. HTMLMediaElement.play() returns a Promise and \`onPlayError\` catches the error if the Promise is rejected. Display the pause control if any error is detected.
+
+If \`autoplay\` is set, don't set the initial \`playing\` state to true as both will attempt to autoplay the video. We recommend setting \`autoplay\`, using \`onPlay\` to detect when the video is played and setting \`playing\` to true. If the initial \`playing\` state is set to true, don't set \`autoplay\`. If both \`autoplay\` and the initial \`playing\` state are set to true, \`autoplay\` has preference.
```

Comment:
I'd argue that yes. 

`autoplay` prop is passed directly to the native `autoplay` attribute. It's purpose is to have autoplay in the video.
`playing` is an abstraction prop. It controls the play/pause toggle in the controls and it can also be used to playback the video on user interaction with the controls (no autoplay). It's purpose is to control play/pause user interactions.